src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;

```

Comment:
The 'quantity' field should be validated to prevent negative values.

Reply:
ok, done